pkg/server/plugin/upstreamca/disk/disk_test.go

```diff
@@ -43,6 +44,11 @@ type DiskSuite struct {
 func (s *DiskSuite) SetupTest() {
 	p := New()
 
+	// This ensures that there are only specific tests that do the verify
+	// flow lowering the cost of "refreshing" all of the cert material
+	// associated with the tests in this package. TODO before 2029 generate
```

Comment:
I'm hoping we get to this sooner than later, using certs in fixtures is smelly :)